src/components/Conversations/ConversationsLayout.tsx

```diff
@@ -0,0 +1,32 @@
+import { ConversationsSidebar } from '~/components/Conversations/ConversationsSidebar';
+import { AppShell, Header, Text } from '@mantine/core';
+
+// TODO: Create conversations provider to maintain states
+export default function ConversationsLayout({ children }: { children: React.ReactNode }) {
+  return (
+    <AppShell
```

Comment:
We're not using the `AppShell` anymore. I'd suggest you go with `<AppLayout>` here. I think it should be pretty much the same you're looking for